static/app/views/settings/dynamicSampling/projectsTable.tsx

```diff
@@ -63,7 +65,7 @@ export function ProjectsTable({
           <IconArrow direction={tableSort === 'desc' ? 'down' : 'up'} size="xs" />
         </SortableHeader>,
         t('Stored Spans'),
-        canEdit ? t('Target Rate') : t('Projected Rate'),
+        canEdit ? t('Target Rate') : t('Estimated Rate'),
```

Comment:
nit: could we replace this with Target Rate (estimated) so the switch is less janky?
```suggestion
        canEdit ? t('Target Rate') : t('Target Rate (estimated)'),
```

Reply:
Actually, it should not switch at all in this case. An oversight from my side. Thx for pointing out!

Reply:
Please not, let's keep "target rate" exclusive to intentionally configured rates and not use that word for estimations or blends.